packages/react-components/src/localization/locales/en-GB/strings.json

```diff
@@ -15,12 +15,17 @@
   },
   "sendBox": {
     "placeholderText": "Enter a message",
-    "textTooLong": "Your message length is over the maximum limit."
+    "textTooLong": "Your message length is over the maximum limit.",
+    "sendButtonAriaLabel": "Send message"
   },
   "messageStatusIndicator": {
+    "deliveredAriaLabel": "Message sent",
     "deliveredTooltipText": "Sent",
+    "seenAriaLabel": "Message seen by others",
     "seenTooltipText": "Seen",
+    "sendingAriaLabel": "Message sending",

```

Comment:
Should this be "Sending message" instead?

Actually, "Sent message", "Failed to send message" might also sound better when announced by the narrator.